peachjam/views/courts.py

```diff
@@ -168,6 +169,12 @@ def get_base_queryset(self, exclude=None):
             qs = qs.filter(court=self.court)
         return qs
 
+    def get_user_follow_form(self):
+        form = UserFollowingForm(
+            initial={"court": self.court.pk, "user": self.request.user.pk}
```

Comment:
user must not be a part of the form

Reply:
we don't need the form in the view, removed